### ISSUE TITLE
podspec not updated when downloading

### ISSUE THREAD

**Issue by SethArchambault** [magrinj/react-native-app-store-review#14]
When I npm the latest version, my podspec looks like this:
https://github.com/magrinj/react-native-app-store-review/blob/release/0.0.3/ios/RNAppStoreReview.podspec

```
[!] The `RNAppStoreReview` pod failed to validate due to 1 error:
    - ERROR | attributes: Missing required attribute `homepage`.
    - WARN  | source: The version should be included in the Git tag.

```

Of course this is fixed on the current master branch - so looks like this just needs to be pushed!

**Comment by magrinj** [comment]
Hi @SethArchambault,

Sorry for the delay, I just published it as 0.0.4 on npm

**Comment by SethArchambault** [comment]
Thanks! :D
